README.md

```diff
@@ -0,0 +1,607 @@
+# Tariff Interpreter
+
+___
+This Interpreter is a reverence implementation for computing a JSON formatted bike-sharing tariff model.
+
+## Contents
+
+___
```

Comment:
I don't like these lines too much :) It could be sufficient to just format the document by using headings and similar semantics. They usually turn out nicely.